src/lines-yardstick.coffee

```diff
@@ -107,3 +103,22 @@ class LinesYardstick
     @rangeForMeasurement.setStart(textNode, startIndex)
     @rangeForMeasurement.setEnd(textNode, endIndex)
     @rangeForMeasurement.getClientRects()[0] ? @rangeForMeasurement.getBoundingClientRect()
+
+  charIndexForScreenPosition: (textNode, targetLeft) ->
+    {textContent: textNodeContent} = textNode
+    rangeRect = null
+    nextCharIndex = -1
+    characterComparator = (char, position, charIndex) =>
```

Comment:
Similar to the comment above, can we inline this comparator? The extra cognitive overhead of jumping to another function doesn't seem required in this case.